test/util/validator.go

```diff
@@ -0,0 +1,79 @@
+package util
+
+import (
```

Comment:
this can also go in the example directory

Reply:
done